public/.env.example.js

```
@@ -20,3 +20,4 @@ var WS_PING_TIMEOUT_PERIOD_MS = 35000
 // var HIDE_RESOURCES = true; // optional
 var DISABLE_PUBLIC_VISIBILITY = false
 var ASK_FOR_TZ_UPDATE = true
+var TOOLTIP_DELAY_MS = 2000
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The var TOOLTIP_DELAY_MS  key has incorrect delimiter

(IncorrectDelimiter)